Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -489,3 +510,45 @@ open class AVFoundationPlayback: Playback {
         Logger.logDebug("destroyed", scope: "AVFoundationPlayback")
     }
 }
+
+// MARK: - DVR
+extension AVFoundationPlayback {
+    open override var minDvrSize: Double {
+        return 60.0
+    }
+
+    open override var usingDVR: Bool {
+        guard playbackType == .live else { return false }
+        guard let currentTime = player?.currentTime().seconds else { return false }
+
+        for range in seekableTimeRanges {
```

Comment:
I think we can inline this _for_ using something like this:

`return seekableTimeRanges.first(where: { $0.timeRangeValue.end.seconds > currentTime})  != nil`

Reply:
Done!